cmd/experimental/README.md

```diff
@@ -0,0 +1,30 @@
+## Experimental integrations
+
+In this folder, you will find integrations of resources with Kueue that are not part of the
+kueue binary.
+
+The Kueue project is not committed to maintaining these integrations long term. They can be used
+as samples you can adapt to support arbitrary resources in Kueue.
+
+Based on user feedback, we might choose to move some of these integrations into the main binary,
+where they will be maintained long term.
+
+### Existing integrations
+
+- TBD
+
+### Adding new integrations
+
+If you would like to contribute with an integration, please start by opening an issue in
+https://github.com/kubernetes-sigs/kueue/issues
+
+Keep in mind the following rules for each integration:
+- It should have it's own `go.mod` and (optionally) `Makefile`.
```

Comment:
I don't think we should provide images. Users should build from source. Let me add it to the README.

Reply:
> I don't think we should provide images.

SGTM
However, I think we should test in CI if Dockerfiles for the experimental integrations can be buildable even if we don't provide the image. Because I don't want to have Dockerfiles that may or may not be buildable.

Reply:
Added.